src/screens/Followings/index.js

```diff
@@ -14,6 +14,7 @@ const Followings = ({dataFollower = [], isLoading, setDataFollower = () => {}})
         isLoadingDiscoveryUser={isLoading}
         showRecentSearch={true}
         withoutRecent={true}
+        isUser={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isUser` is set to `true`, but it's not clear what this prop does from the context provided. Ensure that this prop is properly handled in the `<Followings>` component and its purpose is well-documented for maintainability.
```diff
+         isUser={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->